lib/mail/Mail.php

```diff
@@ -982,6 +985,56 @@ public function getReplyTo()
         return $this->reply_to;
     }
 
+    /**
+     * Add the reply to email address to a Mail object
+     *
+     * @param $list array
+     * both of next formats are supported
+     *  'replytolist' => [
+     *      [
+     *          'email' => 'email1@domain.com',
+     *          'name' => 'name one',
+     *      ], [
+     *          'email' => 'email2@domain.com',
+     *          'name' => 'name two',
+     *      ],
+     *  ],
+     *  'replytolist' => [
+     *      'email1@domain.com',
+     *      'email2@domain.com',
+     *      '',
+     *  ],
+     */
+    public function setReplyToList(array $list)
+    {
+        foreach ($list as $l) {
+            if ($l instanceof ReplyTo ) {
+                $this->reply_to_list[] = $l;
+            } else {
+                if (is_array($l)) {
+                    if (!empty($l) && $l['email'] !== '') {
+                        $this->reply_to_list[] = new ReplyTo($l['email'], $l['name']);
+                    }
+                } else {
+                    if ($l !=='') {
+                        $this->reply_to_list[] = new ReplyTo($l);
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Thanks for the addition! Could you add some unit tests to validate the logic here?

Reply:
added unit tests please check